src/blurhash/__init__.py

```diff
@@ -1,14 +1,21 @@
 from __future__ import absolute_import
 from itertools import chain
 
+
 from PIL import Image
+
 from six.moves import zip
 
-from ._encode import ffi as _ffi, lib as _lib
+from ._functions import ffi as _ffi, lib as _lib
 from ._version import version as __version__
 
 
-__all__ = 'encode', '__version__',
+__all__ = 'encode', 'decode', '__version__',
+
+_pixel_modes = {
+    "RGB" : 3,
+    "RGBA" : 4
+}
```

Comment:
`RGBA` mode just adds another channel with default value `0xff`, some of the use-cases require `RGBA` image so that it would be convenient to some further image processing operations from the pixels obtained, if this is not there, we have to manually apply transforms on `PIL.Image.Image` again, so it would be a helpful feature I thought. (This is just a step during decoding and does not depend on encoded blurhash)